src/components/sections/challenges/TeamChallenge.tsx

```diff
@@ -31,16 +53,20 @@ export default function TeamChallenge(): JSX.Element {
     {
       index: 3,
       title: "Submit!",
-      text: "Once you have completed the challenge, only person needs to submit it at the end of this page",
+      text: "Once you have completed the challenge, only one person needs to submit it at the end of this page",
     },
   ];
 
+  const steps: CardData[] = useMemo(() => {
+    return challenge?.isTeamChallenge ? hackathonChallengeSteps : teamChallengeSteps;
```

Comment:
This is not right I think, all the hackathon challenge are team challenges right? but not all the team challenges are hackathon challenges. 

```suggestion
    return challenge?.isTeamChallenge && !challenge.isHackathonChallenge ? teamChallengeSteps : hackathonChallengeSteps;
```

## Question
I though Gabriella was working on this feature, I remember I had debuging session with her on this same fix. Did you sync?